core/events.go

```diff
@@ -474,6 +474,10 @@ func (em *Events) handlePosEvent(e events.Event) {
 				if !sentMulti {
 					em.lastDoubleClickWidget = nil
 					em.lastClickWidget = up
+					if em.focus != up {
+						em.focusClear() // always any other focus before the click is processed
```

Comment:
This logic isn't supposed to be here anymore, right?

Reply:
good catch!